providers/pool.rb

```diff
@@ -188,7 +188,7 @@ def configure
     configure_application_pool(is_new_auto_start, "autoStart:#{new_resource.auto_start}")
```

Comment:
yeah I wanted to compare the capitalized VALUE with the new value...what is coded there doesn't do anything right...i'll need to take a look at it